doc/using/configuration.chapter.md

```diff
@@ -264,8 +264,9 @@ This provides us with some useful documentation for using our packages.  However
 {
   packageOverrides = pkgs: with pkgs; rec {
     myProfile = writeText "my-profile" ''
-      export PATH=$HOME/.nix-profile/bin:/nix/var/nix/profiles/default/bin:/sbin:/bin:/usr/sbin:/usr/bin
-      export MANPATH=$HOME/.nix-profile/share/man:/nix/var/nix/profiles/default/share/man:/usr/share/man
+      XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+      export PATH=$XDG_DATA_HOME/nix/profile/bin:/nix/var/nix/profiles/default/bin:/sbin:/bin:/usr/sbin:/usr/bin
+      export MANPATH=$XDG_DATA_HOME/nix/profile/share/man:/nix/var/nix/profiles/default/share/man:/usr/share/man
```

Comment:
```suggestion
      XDG_DATA_HOME=${XDG_STATE_HOME:-$HOME/.local/share}
      export PATH=$XDG_STATE_HOME/nix/profile/bin:/nix/var/nix/profiles/default/bin:/sbin:/bin:/usr/sbin:/usr/bin
      export MANPATH=$XDG_STATE_HOME/nix/profile/share/man:/nix/var/nix/profiles/default/share/man:/usr/share/man
```

Reply:
Shouldn't it be `XDG_STATE_HOME=${XDG_STATE_HOME:-$HOME/.local/state}`?